rust/src/mime/smtp.rs

```diff
@@ -584,13 +456,16 @@ fn mime_smtp_parse_line(
                                 for _i in 0..4 - decoder.nb {
                                     v.push(b'=');
                                 }
-                                if let Ok(dec) = mime_base64_decode(decoder, &v) {
+                                let dec_size = ((decoder.nb as usize + v.len()) * 3) / 4;
+                                let mut dec = vec![0; dec_size];
+                                let mut dec_len = 0; // unnecessary but required by fn args
```

Comment:
seems necessary despite the comment as it gets used after

Reply:
yeah i used it to avoid some warning I was getting, I think. But, we can retrieve the dec_len easily w `dec.len()` and that's why I called it unnecessary. wdyt?

Reply:
Are ou sure dec.len() == dec_len ?

For me dec.len() may be bigger

Reply:
Indeed. I misremembered the code from when `dec` was a vector and it was being truncated to the exact decoded bytes. So, the code is correct. I'll remove the misleading comment. Thank you!